src/project/tests/mocks/projectconfigurationmock.h

```diff
@@ -97,6 +97,10 @@ class ProjectConfigurationMock : public project::IProjectConfiguration
     MOCK_METHOD(void, setAutoSaveInterval, (int), (override));
     MOCK_METHOD(async::Channel<int>, autoSaveIntervalChanged, (), (const, override));
 
+    MOCK_METHOD(bool, promptShareAudioCom, (), (const, override));
+    MOCK_METHOD(void, setPromptShareAudioCom, (bool), (override));
+    MOCK_METHOD(async::Channel<bool>, promptShareAudioComChanged, (), (const, override));
```

Comment:
ProjectConfigurationStub also needs to be updated. We use stubs so that we can disable modules in the build parameters